pennylane/gradients/hadamard_gradient.py

```diff
@@ -113,17 +113,18 @@ def hadamard_grad(
     to use during autodifferentiation:
 
     >>> import jax
-    >>> dev = qml.device("default.qubit", wires=2)
+    >>> dev = qml.device("default.qubit")
     >>> @qml.qnode(dev, interface="jax", diff_method="hadamard")
     ... def circuit(params):
     ...     qml.RX(params[0], wires=0)
     ...     qml.RY(params[1], wires=0)
     ...     qml.RX(params[2], wires=0)
-    ...     return qml.expval(qml.Z(0)), qml.var(qml.Z(0))
+    ...     return qml.expval(qml.Z(0)), qml.probs(wires=0)
```

Comment:
Why no variance?

Reply:
It's explicitly not supported, see [here](https://github.com/PennyLaneAI/pennylane/blob/5cc8a622cbb48c6826dd4012730e7f94360dc491/pennylane/gradients/hadamard_gradient.py#L236) :sweat_smile: